lib/hackney/income/tenancy_prioritiser/tenancy_classification.rb

```diff
@@ -70,6 +73,23 @@ def update_court_outcome_action?
           @criteria.court_outcome.blank?
         end
 
+        def breach_letter_action
+          return unless @criteria.most_recent_agreement.present?
+          return unless @criteria.most_recent_agreement[:start_date].present?
+          return unless @criteria.most_recent_agreement[:breached]
+
+          return :send_informal_agreement_breach_letter_new unless @criteria.courtdate.present?
```

Comment:
```suggestion
          return :send_informal_agreement_breach_letter_new if @criteria.courtdate.blank?
```

https://www.rubydoc.info/gems/rubocop/0.53.0/RuboCop/Cop/Rails/Blank